src/graph/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ add_library(
     InterimResult.cpp
     VariableHolder.cpp
     AddHostsExecutor.cpp
-    RemoveHostsExecutor.cpp
+    DropHostsExecutor.cpp
```

Comment:
Keep it as it be until we have a conclusion.

Reply:
I will keep `RemoveHosts` and update if there is progress.